identity/roles.go

```diff
@@ -189,3 +190,117 @@ func (c *RolesClient) Delete(ctx context.Context, input *DeleteRoleInput) error
 
 	return nil
 }
+
+//
+type RoleTags struct {
+	//
+	Name string `json:"name"`
+	//
+	RoleTag []string `json:"role-tag"`
```

Comment:
I wonder if this should be made plural? Looks awkward to have a list there, but singular member name.

Reply:
After talking off-line with @stack72, we've decided to go with a plural form here, despite what the CloudAPI has, as it makes more sense since it is a list of role tags.